digdag-core/src/main/java/io/digdag/core/agent/RequireOperatorFactory.java

```diff
@@ -103,5 +103,34 @@ public TaskResult runTask()
                 throw new TaskExecutionException(ex);
             }
         }
+
+
+        /**
+         * Make ProjectIdentifier from parameters.
+         * @return
+         */
+        private ProjectIdentifier makeProjectIdentifier()
+        {
+            Config config = request.getConfig();
+            Config localConfig = request.getLocalConfig();
+            int projectId = config.get("project_id", int.class);
+            Optional<Integer> projectIdParam = localConfig.getOptional("project_id", Integer.class);
+            Optional<String> projectNameParam = localConfig.getOptional("project_name", String.class);
+
+            if (projectIdParam.isPresent() && projectNameParam.isPresent()) {
```

Comment:
Do you think it's reasonable to do the same check in compile time when a workflow is pushed?

Reply:
Hmm I am not sure but I will check it later.
If require> has already validation in compile, I will add it. But if not, it may be another improvement in my understand. I will check it later.

Reply:
Currently it seems no validation on operators in check. So I would like to skip it.